magni_description/urdf/magni.urdf.xacro

```diff
@@ -153,18 +153,31 @@
     </xacro:sonar_hc-sr04>
   </xacro:if>
 
-
   <!-- Define raspicam based on the raspicam_mount argument -->
+  <!-- forward  Legacy tilted slightly upwards              -->
+  <!-- upward   Legacy pointing straight up                 -->

```

Comment:
upward is not legacy

Reply:
Why is 'upward' not legacy?  Our bots have always had the upward camera and I thought that was a key and important feature.   granted it doe not nav well yet.

Are you saying it was there but never supported in any way????

Reply:
To me legacy implies 'we don't recommend this anymore, use new stuff'. Basically the step before deprecation.

Reply:
You may be able to make the case for 'forward' because the name is not accurate for legacy reasons, but upward is a current supported/recommended configuration with an accurate name.